OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -164,6 +168,8 @@ void CaptureClient::FinishCapture() {
   grpc::Status status = reader_writer_->Finish();
   if (!status.ok()) {
     ERROR("Finishing gRPC call to Capture: %s", status.error_message());
+    capture_listener_->OnCaptureFailed(
+        ErrorMessage(absl::StrFormat("Error while finishing capture: %s", status.error_message())));
```

Comment:
I think we want to not report this. In one case it was already reported in another - it is not really relevant since the capture was already complete.

Reply:
Okay, I'll remove this call